app/assets/javascripts/admin/templates/components/admin-report.hbs

```diff
@@ -173,6 +173,17 @@
           </div>
         {{/if}}
 
+        {{#each filterOptions as |filterOption|}}
```

Comment:
```
              {{combo-box content=filterOption.choices
                          filterable=true
                          value=filterOption.selected
                          allowAny=true
                          onSelect=(action "filter" filterOption.id)}}
```

I suggest we try this, so we can filter by any extension if needed, eg: mp4 used to be allowed before and I want to find them